server/repository/src/migrations/v2_06_03/remove_non_custom_standard_reports.rs

```diff
@@ -0,0 +1,25 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "remove_non_custom_standard_reports"
+    }
+
+    // For non-transfer-related processing of requisitions
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        if cfg!(feature = "postgres") {
+            sql!(
+                connection,
+                r#"
+                    DELETE FROM changelog WHERE record_id IN 
+                    (SELECT id FROM report WHERE is_custom = false AND context = 'REPORT');
+                    DELETE FROM report WHERE is_custom = false AND context = 'REPORT';
```

Comment:
Do you think we will get the same problem with custom reports if upgrading?

Reply:
We could generate lots of sync traffic with custom reports, especially if we upsert a bundle that uses extism, but this should have been removed now, hopefully and report sync was only added in 2.6